packages/web/index.html

```diff
@@ -246,6 +246,7 @@ <h3>Thing Description Playground</h3>
                       <button type="button" class="btn custom-btn" id="btn_gistify"
                         title="Stores the current TD/TM as a gist at Thingweb">Gistify</button>
                       <button type="button" class="btn custom-btn" id="btn_assertion_popup">Assertion Test</button>
+                      <button type="button" class="btn custom-btn" id="btn_save">Save</button>
```

Comment:
I think this might be a bit confusing for some privacy aware people since we are actually not saving the td (in comparison to the two tools linked in the issue). We are giving them a URL that they can save somewhere themselves. Editdor uses "share" but I find it a bit weird since that is associated (at least for me) with smartphones that show apps that can share it. Not sure how to really call it but some ideas (more are welcome):
- Generate URL
- Copy as URL
- Get Link or just Link

In all cases, at least a hover mechanism should be used to tell what will happen to the user

Reply:
Better now?

Reply:
Good now?